src/widgets/pixbuf_loader.rs

```diff
@@ -43,7 +43,7 @@ impl PixbufLoader {
     ///
     /// # Failures
     /// Returns `None` if the pixbuf loader cannot be created.
-    pub fn new_with_type(image_type: &str, error: &mut ::glib::Error) -> Option<PixbufLoader> {
+    pub fn new_with_type(image_type: &str, error: &mut Error) -> Option<PixbufLoader> {
```

Comment:
I mentioned that in #15:  https://github.com/rust-gnome/gdk/pull/15#issue-72717508
What are your thoughts?